src/app/modals/share-link-modal/share-link-modal.component.pug

```diff
@@ -27,5 +27,5 @@
               | {{ 'SHARE_IMG_LINK_MODAL.COPY_TO_CLIPBOARD' | translate }}
             .copy-status-msg {{ copyHTMLStatusMsg }}
       .modal-footer
-        button.btn.btn-link.with-pad((click)="showHelp()", tabindex="0") {{ 'SHARE_IMG_LINK_MODAL.HELP' | translate }}
```

Comment:
Didn't really see the need for a function to handle linking to another page. I converted it to a link, let me know if I could be missing something though